requirements/_.txt

```diff
@@ -1,2 +1,2 @@
 aiohttp>=3.6.0,<4.0
-typing_extensions>=4,<5; python_version < "3.11"
+typing_extensions>=4,<5
```

Comment:
This will result in a swarm of people complaining about a `ModuleNotFoundError`, therefore I'm not sure if this is the right time to make this change. We could wait till v3.

Reply:
I'm not sure to get what you mean - if they update, new dependencies should be also automatically added ?

Reply:
But maybe better for v3, idk. Only things is that I used typing_extensions in a bunch of other prs (added to reqirements too).

Reply:
> if they update, new dependencies should be also automatically added ?

Oh yeah, right. Installing the new release should install typing_extensions too. My bad.

Reply:
Besides, this shouldn't cause any deps conflicts as I believe some py-cord sub dependency requires a compatible version of typing_extensions.